openbas-model/src/main/java/io/openbas/database/model/Inject.java

```diff
@@ -421,45 +421,80 @@ public static Inject fromRawInject(RawInject rawInject,
 
     // Set a list of expectations
     inject.setExpectations(new ArrayList<>());
-    for(String expectationId: rawInject.getInject_expectations()) {
-      // Create a new expectation
-      InjectExpectation expectation = new InjectExpectation();
-      expectation.setId(rawInjectExpectationMap.get(expectationId).getInject_expectation_id());
-      expectation.setType(InjectExpectation.EXPECTATION_TYPE.valueOf(rawInjectExpectationMap.get(expectationId).getInject_expectation_type()));
-      expectation.setScore(rawInjectExpectationMap.get(expectationId).getInject_expectation_score());
-
-      // Add the team of the expectation
-      Team team = new Team();
-      team.setId(rawInjectExpectationMap.get(expectationId).getTeam_id());
-      team.setName(rawTeams.get(rawInjectExpectationMap.get(expectationId).getTeam_id()).getTeam_name());
-      expectation.setTeam(team);
+    for (String expectationId : rawInject.getInject_expectations()) {
```

Comment:
Personnaly I think this method is too long, you should split it if it is possible